examples/index.js

```diff
@@ -0,0 +1,41 @@
+const Fastify = require('fastify')
+
+async function run({ token, ...options }) {
+  if (!token) {
+    throw new Error('Please provide a token via --token {token}')
+  }
+
+  const fastify = Fastify()
+
+  await fastify.register(require('..'), options)
+
+  fastify.get(
+    '/',
+    {
+      preValidation: fastify.authenticate
+    },
+    r => r.user
+  )
+
+  await fastify.listen()
```

Comment:
I think listen is not needed if you're just using inject.

Reply:
Confirmed. I tested it locally. You can remove it.

Reply:
You're right, done